build.gradle.kts

```diff
@@ -7,6 +7,10 @@ plugins {
     alias(libs.plugins.kotlinxKover) apply false
 }
 
+apply {
+    from("${rootDir}/git-hooks.gradle.kts")
+}
+
```

Comment:
Please define the install task directly.

```suggestion
```

Reply:
Any reason for defining it directly?
I kept a separate file for clean code (Separation of Concerns) and better visibility.

Reply:
I think you can refer to this.
https://developer.squareup.com/blog/herding-elephants/#model-your-build